alpine/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG ARCH=""
+FROM        ${ARCH}alpine:latest as alpine
+
+RUN apk upgrade -U && \
+    apk add tzdata ca-certificates
+
+# Use the busybox.static to avoid dynamic library dependencies.
+RUN apk add busybox-static && \
+    mv /bin/busybox.static /bin/busybox && \
+    /bin/busybox --install -s /bin
+
+# remove extra package
+RUN apk del libc-utils
+
+# remove apk files and directories
+RUN apk del apk-tools && \
+    find / -name apk -prune -exec rm -rf {} ";"
```

Comment:
Combine these steps to reduce layers.

Reply:
We don't need to worry about these layers because the `COPY` on line 26 flattens all the build layers to a single layer in the final image.